universal_transfer_operator/transfers/data_providers/__init__.py

```diff
@@ -0,0 +1,43 @@
+import importlib
+import os.path
+
+from transfers.constants import TransferMode
+from transfers.data_providers.base import DataProviders, TempFile
+from transfers.datasets.base import UniversalDataset as Dataset
+
+from astro.constants import LoadExistStrategy
+from astro.utils.path import get_class_name
+
+DATASET_CONN_ID_TO_DATAPROVIDER_MAPPING = {
+    "s3": "data_providers.filesystem.aws.s3",
+    "aws": "data_providers.filesystem.aws.s3",
+    "gs": "data_providers.filesystem.google.cloud.gcs",
+    "google_cloud_platform": "data_providers.filesystem.google.cloud.gcs",
+}
+
+
+def create_dataprovider(
+    dataset: Dataset,
+    transfer_params: dict = {},
+    transfer_mode: TransferMode = TransferMode.NONNATIVE,
+    if_exists: LoadExistStrategy = "replace",
```

Comment:
should be part of `transfer_params`

Reply:
https://github.com/astronomer/astro-sdk/pull/1492/commits/22b0aeeb515b6e9eb72dd68f62a64cb807a5732a